test/opus/pipeline/stage/skip_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule Opus.Pipeline.Stage.SkipTest do
+  use ExUnit.Case
+
+  describe "when the stage returns false and there's no next stage" do
```

Comment:
We should add an example about what happens if there are multiple skips.

```elixir
defmodule SomePipeline do
  skip if: :skip?
  skip if: :please_skip?
  skip if: :should_it_skip_maybe?

  step :some_step
end
```

Reply:
Sure! I'll add it

Reply:
Added three more cases about multi skips, take a look